fandogh_cli/image_commands.py

```diff
@@ -62,13 +62,22 @@ def list_images():
 
 def show_image_logs(image_name, version):
     image_offset = 0
+    counter = 0
 
     if not image_name:
         image_name = get_project_config().get('image.name')
     while True:
         response = get_image_build(image_name, version, image_offset)
+
+        if response.get('state') == 'PENDING' and \
```

Comment:
why is this check is needed?

Reply:
you probably just need to update line 83 to check state is not bulilding or pending.

Reply:
Good point